src/components/Sidebar/Sidebar.tsx

```diff
@@ -179,14 +180,19 @@ const Sidebar: React.FC<RouteComponentProps & SidebarPropsType> = ({ location, .
                 )}
               </React.Fragment>
             ))}
+            <Hidden mdUp>
+              <ListItem style={{ justifyContent: 'flex-end' }}>
+                <SwitchMode />
+              </ListItem>
+            </Hidden>
           </Styles.Items>
         </List>
       </Styles.Scrollbar>
       <Styles.SidebarFooter>
         <Styles.SidebarContainer container spacing={2} justify="space-around" alignItems="center">
           {footerIcons.map(({ id, url, icon }) => (
             <Grid item key={id}>
-              <IconButton target="_blank" href={url} color="primary" style={{ padding: '4px' }}>
+              <IconButton target="_blank" href={url} style={{ color: '#6151d0', padding: '4px' }}>

```

Comment:
If it's complex to use color from theme, consider defining and using css variables, they can be changed by dark mode as well.
But I guess theme should be enough for managing colors, isn't it?